DigitalLearningSolutions.Web/Controllers/LoginController.cs

```diff
@@ -81,6 +127,7 @@ private void LogIn(AdminUser? adminUser, DelegateUser? delegateUser, bool rememb
                 IssuedUtc = DateTime.UtcNow
             };
             HttpContext.SignInAsync("Identity.Application", new ClaimsPrincipal(claimsIdentity), authProperties);
+            return RedirectToAction("Index", "Home");

```

Comment:
Is there a reason behind this change (now the login method logs the user in and redirects to home page)?

Reply:
I just thought that because the LogIn method will be called from the single centre log in and also the choose a centre log in that both send the user to the same place, I might as well save a line of code and bung it in there. Would we prefer the extra specificity of before where each method returning specific actions rather than having to look through the other methods it calls?

Reply:
I think it's ok actually as I don't think the redirects should ever be different between those pages.